src/main/webapp/creatives/maps/utils.js

```diff
@@ -1,13 +1,13 @@
 
 /** A list of server routes that keep track of each interaction when fetched. */
 export const URL = {
```

Comment:
This seems like it belongs more in a const file then a utils file.
Also, `URL` should be `Urls`.  Its not CAPITAL_CASE since it is an object, so its not deeply immutable.  The properties of it however are deeply immutable and can stay CAPITALIZED
https://google.github.io/styleguide/jsguide.html#naming-constant-names